climakitae/explore/agnostic.py

```diff
@@ -277,20 +277,21 @@ def create_conversion_function(lookup_tables):
 ##### TASK 2 #####
 
 
-def _get_var_info(variable, downscaling_method):
+def _get_var_info(variable, downscaling_method, wrf_timescale="monthly"):
     """Gets the variable info for the specific variable name and downscaling method"""
     var_desc_df = read_csv_file(variable_descriptions_csv_path)
+    timescale = wrf_timescale if downscaling_method == "Dynamical" else "monthly"

```

Comment:
I think this line is ultimately redundant, especially since the default setting is intentionally "monthly". We've included options on how to set this in the notebook

Reply:
I think this line just prevents LOCA2 simulations from being run on more granular timescales (like daily or hourly), in case the user were to input daily/hourly for `wrf_timescale` but also use a `downscaling_method` of Statistical.